domain/src/main/java/org/oppia/android/domain/oppialogger/analytics/CpuPerformanceSnapshotter.kt

```diff
@@ -24,12 +30,13 @@ class CpuPerformanceSnapshotter(
   private val initialIconification: AppIconification,
   private val consoleLogger: ConsoleLogger,
   private val exceptionLogger: ExceptionLogger,
+  private val performanceMetricsAssessor: PerformanceMetricsAssessor,
   private val foregroundCpuLoggingTimePeriodMillis: Long,
   private val backgroundCpuLoggingTimePeriodMillis: Long,
-  private val performanceMetricsAssessor: PerformanceMetricsAssessor
+  private val initialIconificationCutOffTimePeriodMillis: Long
 ) {
 
-  private val commandQueue by lazy { createCommandQueueActor() }
+  private val commandQueue = createCommandQueueActor()
```

Comment:
Why this change? I'm not opposed, but the idea with ``lazy`` is it helps us avoid making constructors computationally expensive (which is a Dagger antipattern for injectable classes).

Reply:
I thought of removing the lazy implementation completely and introduce an initialisation method. It was based on the assumption that we can rule out `commandQueue` and use actor channels instead. But it wasn't entirely possible since the `commandQueue` was getting used in methods that are not present in the actor and hence don't have access to the channel instance -- for example: `sendSwitchIconificationCommand` in `updateAppIconification` doesn't have access to the channel and hence can't use the same actor instance to perform the update.